webapp/src/components/github_issue/github_issue.jsx

```diff
@@ -0,0 +1,117 @@
+import {makeStyleFromTheme} from 'mattermost-redux/utils/theme_utils';
```

Comment:
i think this has been discussed before whenever we create a new file that should be .ts and new component should be functional component. Is there any reason why this file .js ?